fuzz/wrap.c

```diff
@@ -16,6 +16,7 @@
 #include <fido.h>
 
 #include "drop_privs.h"
+#include "cfg.h"
```

Comment:
What are we using this header for here?

Reply:
Left-over from first attempts. To be dropped.